tools/buildsteps/windows/x64-uwp/default.bat

```diff
@@ -0,0 +1,23 @@
+@echo OFF
+
+pushd %~dp0\..
+call .helpers\default.bat
+call vswhere.bat x64 store
+if ERRORLEVEL 1 (
+	echo ERROR! Something went wrong when calling vswhere.bat
+	popd
+	exit /B 1
+)
+popd
+
+if "%Configuration%"=="" (
+	set BUILD_TYPE=Debug
+) else (
+	set BUILD_TYPE=%Configuration%
+)
+
+set TARGET_CMAKE_GENERATOR=Visual Studio %vsver%
+set TARGET_CMAKE_GENERATOR_PLATFORM=x64
+set TARGET_CMAKE_OPTIONS=-DCMAKE_SYSTEM_NAME=WindowsStore -DCMAKE_SYSTEM_VERSION=%UCRTVersion%
```

Comment:
No, that would change behaviour.

Reply:
change what behaviour? it already picks some version, does it matter if it's vcvarsall.bat or cmake that decides?

Reply:
No it doesn't pick a random version, at least for platforms targeting WindowsStore.

https://github.com/xbmc/xbmc/blob/952675dcf92d564840c4b5560f02340d4ef52a1f/tools/buildsteps/windows/vswhere.bat#L40-L41

Reply:
`CMAKE_SYSTEM_VERSION=10.0` will pick up the latest installed SDK, which can break the build of dependencies such like OpenSSL

Reply:
ahh yeah, my bad. I guess we could give the full version we require directly to cmake instead of running vswhere.
The one benefit of vswhere I can see is that it can check for installed VS versions if we want to support both vs2017 and vs2019. We should remove the check for vs2015 from it but that's for another PR.

Reply:
Some cmake versions (e.g. 3.13) is broken if you specify the full Windows SDK version and use the VS2019 compilers.

Reply:
Thanks for the info, but it's not relevant for now, as currently only `Visual Studio 2017` is supported.

Reply:
@aronyu79 It doesn't work for uwp only iirc